App.tsx

```diff
@@ -33,7 +33,7 @@ export default class App extends React.PureComponent {
     return (
       <I18nextProvider i18n={i18n}>
         <Provider store={store}>
-          <AppStateManager handleAppComesToBackground={this.lockScreen} />
+          <AppStateManager handleAppComesToForeground={this.lockScreen} />
```

Comment:
what is the reason for this change? with this, you can hack the pin screen by clicking real fast a tab bar when app is foregrounding

Reply:
It was a quick workaround for the beta, but ok, I will sit to it once more